docs/about/features.md

```
@@ -9,39 +9,44 @@ description: Learn how CodeRabbit works in a pull request review.
 that is part of the pull request, followed by a celebratory poem about the
 changes.
 
-![Summary Walkthrough](/img/about/Summary-Walkthrough.png)
+![CodeRabbit- Pull Request Walkthrough Example](/img/about/Summary-Walkthrough.png)
 
 ### 2. Code Review feedback {#code-review-feedback}
 
-Review feedback is posted as review comments on the lines of the code that
-changed for each file. Code suggestions are given in a Diff format, which either
+**Code review** feedback is posted as review comments on the lines of the code that
+changed for each file. 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)